### PR TITLE
Set ForceAuthn property on (certain) outgoing AuthNRequests

### PR DESCRIPTION
**Pull request by MKodde**
The new saml bundle is used to be able to set the force authn property on the authentication requests being sent to the Azure MFA IdP.

The registration.feature and authentication.feature are piggybacked to verify the ForceAuthn property is set on the outgoing AuthNRequest.

In order to get this PR to pass CI quality checks I had to:
1. Upgrade the ChromeDriver version
2. Update composer dependencies
3. Repair PHP MD issues after PHP MD was updated.

### CODE REVIEW

**Comment by pablothedude** [comment]
Please elaborate why the composer dependencies were updated.

Maybe we also have to come up with a better solution for the Chromedriver altogether or maybe we should stop using it. But that's out-of-scope for now.

**Comment by MKodde** [comment]
> Please elaborate on why the composer dependencies were updated.

Added additional comment to the commit where the composer update results where committed. In short, it's just updating for the sake of staying up to date.

And on a more serious note. This PR was changed slightly to facilitate an additional feature request on the story. This addition adds the possibility to configure a regex, that when matches, will add force authn property to outgoing requests.

**Comment by MKodde** [comment]
> I was wondering if it wouldn't have been sufficient or even better to just always use force authn for an Azure GSSP callout. If someone else is logged in in an AzureIdP would you then be unable to use your identity as a second factor for your account?

I don't think that would result in a usable second factor provider. Imagine every time you do a Stepup-stepup action you would have to authenticate at the Azure MFA, and then give a step up authenticadtion there. But maybe I see this the wrong way around. Maybe @phavekes can join in on this discussion? And we shall have a word in real life about this! 

> 

And I still don't like the force_auth parameter slowely being used all over the place. Instead of being injected from the config once into the factory.

That would not have worked in this use case. As the backend IdP's are not to be configured with this config. The issuing SP would be, and that is not part of the config options.

File: config/packages/parameters.yaml.dist [hunk 1]
```diff
@@ -18,4 +18,5 @@ parameters:
     saml_remote_sp_sso_url: '"https://pieter.aai.surfnet.nl/simplesamlphp/module.php/saml/sp/saml2-acs.php/default-sp"'
     saml_remote_sp_certificate: '%kernel.root_dir%/../../../../vendor/surfnet/stepup-gssp-bundle/src/Resources/keys/pieter.aai.surfnet.nl.pem'
     saml_remote_sp_acs: 'https://pieter.aai.surfnet.nl/simplesamlphp/module.php/saml/sp/saml2-acs.php/default-sp'
-
+    # Authentication issuers matching this regex will send ForceAuthn to the Azure MFA. The at sign is used as delimiter, be sure to escape the first sign as Symfony would see it as a reference to a service.
+    ra_issuer_entity_id_regex: '@@^https://(?:[a-z0-9](?:[a-z0-9-]{0,61}[a-z0-9])?\.)+[a-z0-9][a-z0-9-]{0,61}[a-z0-9]/vetting-procedure/gssf/azuremfa/metadata$@'
```
[the remark this answers is not in the document]
Comment (MKodde) [reply config/packages/parameters.yaml.dist]: That might be possible, and can certainly be done here. For now I'm leaving it as is.

**Pull request merged**